source/background.ts

```diff
@@ -45,6 +45,8 @@ addDomainPermissionToggle();
 
 // `background` fetch required to avoid avoid CORB introduced in Chrome 73 https://chromestatus.com/feature/5629709824032768
 // Don’t turn this into an `async` function https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessage#addListener_syntax
-browser.runtime.onMessage.addListener(({request}): Promise<Response> | void => {
-	return fetch(request).then(async response => response.json());
+browser.runtime.onMessage.addListener((message): Promise<string> | void => {
+	if (message?.request) {
+		return fetch(message.request).then(async response => response.text());
```

Comment:
Does this fail lint? I don't remember if we disabled that rule
```suggestion
		return fetch(message.request).then(response => response.text());
```

Reply:
No we kept is since it was only one place